services/app-api/utils/actionDelegate.js

```diff
@@ -109,6 +112,10 @@ export function getActionsForPackage(
         ),
         ...getWaiverExtensionActions(packageStatus, userRole)
       );
+      //Extensions should remove SUBSEQUENT_SUBMISSION action
+      actions = actions.filter(
```

Comment:
true, i can remove if we feel that is better than trying to worry about what might happen in the future......i sometimes overreach like that wanting to cover changes dynamically

Reply:
Which is an admirable trait, for sure!  I don't think it needs to be removed, since it *does* have a specific AC regarding it, so kinda makes sense that the possibilities are covered, even if they are not currently relevant.